commons/dev/configmaps/flyway-readmodel-tenant-configmap.yaml

```diff
@@ -197,3 +197,7 @@ data:
   V1.3.10__Grant_Access_SelfcareClientUsersUpdater_Tenant.sql: |-
     GRANT USAGE ON SCHEMA "${NAMESPACE}_tenant" to "${NAMESPACE}_selfcare_client_users_updater_user";
     GRANT SELECT ON ALL TABLES IN SCHEMA "${NAMESPACE}_tenant" TO "${NAMESPACE}_selfcare_client_users_updater_user";
+
+  V1.3.11__Grant_Access_NotificationConfigProcessUser_Tenant.sql: |-
```

Comment:
this is not to deploy the BFF actually, but to add a permission that I should have added in https://github.com/pagopa/interop-core-deployment/pull/270 and which currently means some notification-config-process routes don't work in dev (the ones that need to read from the tenant readmodel)